style.css

```diff
@@ -655,6 +687,221 @@ footer {
   width: 1px;
 }
 
+/* ///////Popup//////// */
+
+.popup {
+  width: 97%;
+  height: fit-content;
+  position: absolute;
+  top: 0;
+  left: 0;
+  margin: 0 auto;
+  float: left;
+  padding: 3% 1.5%;
+  display: none;
+}
```

Comment:
- [ ] Great job on rendering the projects list dynamically :+1: However, Kindly consider making sure when the pop modal is activated the user can't scroll anymore to overpass it as is the case on your design.
Hint: `to fix this issue, you may consider adding a container  for the pop-up in the HTML doc, give it the absolute position according to the body element, and center it as you did for the pop up and I hope from there you will be good to go.`

### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/224162515-0cf51e31-cde1-4705-b069-8a3c01002b99.png)
![image](https://user-images.githubusercontent.com/97100091/224162586-c0dea2ae-59ad-467c-b9d1-5b8c317da0cc.png)